File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 9/14/2025, 9:15:04 AM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        j=0
        for i in nums:
            if j!=i:
                return j
            j+=1
        return j

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 9/14/2025, 9:14:57 AM
class Solution {
    public int missingNumber(int[] nums) {
        int xor = 0;
        // int xor2 = 0;
        for(int i=0; i<nums.length; i++){
            xor ^= i ^ nums[i];
        }
        xor ^= nums.length;

        return xor;
    }
}